src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -1295,13 +1295,17 @@ def aks_get_credentials(client, resource_group_name, name, admin=False,
                 pass
 
         # merge the new kubeconfig into the existing one
-        with tempfile.NamedTemporaryFile(mode='w+t') as additional_file:
+        fd, temp_path = tempfile.mkstemp()
+        additional_file = os.fdopen(fd, 'w+t')
```

Comment:
`t` is not necessary as `text` mode is the default, but if you prefer explicit, that is fine